tests/test_p4_submit.py

```diff
@@ -37,3 +38,23 @@ def test_success_changing_checked_out_file(p4_submit_environment):
     p4_submit_environment.p4.run_submit(change)
 
     assert p4_submit_environment.file_present(str(target_file))
+
+
+@pytest.mark.parametrize("test_type,expected",
+                         [("open", 0), ("write", 1), ("review", 1)],
```

Comment:
"review" doesn't describe the protection exactly.
I would say it is trigger-protected branch.